manifests/init.pp

```diff
@@ -8,8 +8,8 @@
 class profile_session_log (
   Boolean $enable_session_log,
```

Comment:
That name of this variable seems a bit redundant.

Any reason to not simply call it `profile_session_log::enable` ?